manila/README.md

```diff
@@ -0,0 +1,194 @@
+# Dell EMC Manila driver containerization with OSP13
+
+## Overview
+
+Dell EMC Manila driver integrates with the RedHat OpenStack Platform since OSP 6. In OSP13, all Manila services are able to run within containers.
+
+This instruction provides detailed steps on how to enable the containerization of VNX and Unity Manila driver on top of the OSP Manila images.
+
+The Dell EMC Manila container image contains following RPM packages:
+
+- python-storops
+- python-persist-queue
+- python-cachez
+- python-retryz
+- EMC Naviseccli
```

Comment:
We don't need naviseccli for manila driver, do we?

Reply:
No, we don't, removed this line.